src/Tribe/Tickets_View.php

```diff
@@ -962,33 +962,32 @@ protected function normalize_rsvp_option( &$option ) {
 	 *
 	 * @since 4.11.0
 	 * @since 4.12.3 Update usage of get_event_ticket_provider().
+	 * @since TBD Moved getting of context array to its own method.
```

Comment:
This is useful, but it's late in the sprint to make this change, we need to workaround it for now until a future sprint